ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -339,6 +343,9 @@ sysctl -w net.ipv6.conf.\$(ip -o route get 1.1.1.1 | cut -f 5 -d ' ').accept_ra=
 
 cd dev-scripts
 
+git fetch https://github.com/derekhiggins/dev-scripts cs9
```

Comment:
just for testing?

Reply:
yup, I'll remove it once this merges https://github.com/openshift-metal3/dev-scripts/pull/1664